### ISSUE TITLE
using Codext guess / Codext rank gives an error

### ISSUE THREAD

**Issue by GitSunburn** [dhondta/python-codext#5]
When I run it on linux and try to use "codext guess" or "codext rank" I get an error message saying:

```sh
# echo "3yZe7d" | codext rank
Traceback (most recent call last):
  File "/usr/local/bin/codext", line 8, in &lt;module&gt;
    sys.exit(main())
  File "/usr/local/lib/python3.9/dist-packages/codext/__init__.py", line 184, in main
    args.include, args.exclude = __format_list(args.include), __format_list(args.exclude, False)
AttributeError: 'Namespace' object has no attribute 'include'
```

**Comment by dhondta** [comment]
Hi @GitSunburn !
Thank you for mentioning this.
This is typically due to a missing argument. This issue was fixed in a later version.
Please `pip install --upgrade codext`. Then you should see version 1.13.4 with `pip show codext`.

**Comment by fluk33** [comment]
I'm getting the same error with v1.14.0

`$ codext -i user02/user02.txt guess                                                                                                                                                                                                            
Traceback (most recent call last):
  File "/home/nstewart/.local/bin/codext", line 8, in <module>
    sys.exit(main())
  File "/home/nstewart/.local/lib/python3.10/site-packages/codext/__init__.py", line 252, in main
    raise e
  File "/home/nstewart/.local/lib/python3.10/site-packages/codext/__init__.py", line 228, in main
    s, lb = args.stop_function, args.lang_backend
AttributeError: 'Namespace' object has no attribute 'lang_backend'`

**Comment by dhondta** [comment]
@fluk33 Your issue is solved in #7
